contrib/spec/podman.spec.in

```diff
@@ -39,7 +39,7 @@
 %global shortcommit_conmon %(c=%{commit_conmon}; echo ${c:0:7})
 
 Name: podman
-Version: 1.6.0
+Version: 1.6.0-rc2
```

Comment:
@lsm5 @baude Is this OK? I just used the release script and it doesn't seem to handle the `-rc` part well